test/utils/viewHelpers/AtedViewSpec.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.viewHelpers
+
+import java.util.UUID
+
+import builders.AuthBuilder.{createAtedContext, createUserAuthContext}
+import models.AtedContext
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatestplus.play.{OneAppPerSuite, PlaySpec}
+import play.api.{Application, Mode}
+import play.api.i18n.Messages
+import play.api.inject.guice.{GuiceApplicationBuilder, GuiceableModule}
+import play.api.test.FakeRequest
+import play.twirl.api.Html
+
+trait AtedViewSpec extends PlaySpec
+  with JsoupMatchers
+  with OneAppPerSuite {
```

Comment:
why have changed from OneServerPerSuite?

Reply:
OneServerPerSuite is deprecated so using GuiceOneAppPerSuite